### PR TITLE
removing some redundant str splits

### PR DESCRIPTION
**Pull request by Borda**
drop some redundant string splits, no factual changes, just cleaning the codebase

### CODE REVIEW

**Comment by mikaylagawarecki** [comment]
@pytorch rebase

(rebasing as it looks like there is some duplication with https://github.com/pytorch/pytorch/pull/106013)

**Review by albanD — APPROVED** [review]

**Review by malfet — APPROVED** [review]

**Comment by malfet** [comment]
I'll fwd fix lint failures when this PR lands

**Comment by albanD** [comment]
@Borda could you rebase on latest main and fix the lint please?

**Comment by albanD** [comment]
@pytorchbot merge

Thanks!

**Comment by albanD** [comment]
Ho the lint is still not ok :/

**Comment by Borda** [comment]
> Ho the lint is still not ok :/

is there a log I could check because to mee all seems fine...

**Comment by Borda** [comment]
This link complains looks the same to me... :thinking: 
```
    2026  2026 |     param_to_param_names = _get_param_to_fqns(model)
    2027  2027 |     for param_names in param_to_param_names.values():
    2028  2028 |         assert (
    2028       |-len(param_names) > 0
          2029 |+len(param_names) > 0
    2030  2030 |         ), "`_get_param_to_fqns()` should not construct empty lists"
    2031  2031 |         if len(param_names) > 1:
    2032  2032 |             raise RuntimeError(

**Comment by mikaylagawarecki** [comment]
@borda this is what I see on the CI logs [here](https://hud.pytorch.org/pr/106089). Seems like it is complaining about the lack of spaces.

Screenshot 2023-08-30 at 4 26 24 PM


To fix the failing lint job running `setup_lint` and `quicklint` from your local pytorch directory and following the corresponding instructions (running lintrunner -a to fix the remaining failures) should be sufficient, see [here](https://github.com/pytorch/pytorch/blob/4721553431e968ee5e90753346d92252cce98e84/Makefile#L25-L34)

**Comment by Borda** [comment]
> this is what I see on the CI logs [here](https://hud.pytorch.org/pr/106089), but it seems like you see something different locally (?). Seems like it is complaining about the lack of spaces.

o, then there is some issue with how the logs are passed to the GH check at the bottom of a PR because this is what you can find there:
image

File: torch/distributed/tensor/parallel/api.py [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Copyright (c) Meta Platforms, Inc. and affiliates
+ # Copyright (c) Meta Platforms, Inc. and affiliates
```
Comment (awgu) [inline torch/distributed/tensor/parallel/api.py:1]: Curious: Is this added space intentional?

Comment (Borda) [inline torch/distributed/tensor/parallel/api.py:1]:
```suggestion
# Copyright (c) Meta Platforms, Inc. and affiliates
```

File: torch/utils/collect_env.py [hunk 2]
```diff
@@ -165,7 +165,7 @@ def get_cudnn_version(run_lambda):
         # Use CUDNN_LIBRARY when cudnn library is installed elsewhere.
         cudnn_cmd = 'ls /usr/local/cuda/lib/libcudnn*'
     else:
-        cudnn_cmd = 'ldconfig -p | grep libcudnn | rev | cut -d" " -f1 | rev'
+        cudnn_cmd = 'ldconfig -p | grep libcudnn | rev | cut -d -f1 | rev'
```
Comment (albanD) [inline torch/utils/collect_env.py:168]: This does NOT look right!

**Review by albanD — CHANGES_REQUESTED** [review]
I think the one in collect_env is not ok. Rest looks good.

File: torch/utils/collect_env.py (hunk 2 above)
Comment (Borda) [reply torch/utils/collect_env.py:168]: ahh, my mistake, thx
```suggestion
        cudnn_cmd = 'ldconfig -p | grep libcudnn | rev | cut -d" " -f1 | rev'
```

**Review by albanD — APPROVED** [review]
Thanks!

**Pull request closed without merging**